cdap-ui/app/features/admin/controllers/namespace/settings-ctrl.js

```diff
@@ -45,10 +45,19 @@ angular.module(PKG.name + '.feature.admin')
 
     };
 
+    /**
```

Comment:
One downside I see to this approach is that this requires that the UI manages some aspects of namespace deletion.
For instance, if a user deletes a namespace from the CLI or via `curl`, these things will not get cleaned up, and so they will be visible the next time the namespace is created.
Perhaps a namespaced-storage can be used instead, and the deletion of the namespaced configurations would be managed by the namespace delete action in the backend.
@bdmogal Any suggestions?